check-copyrights.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Get all golang files in the repo, excluding the vendor dir.
+files=$(find . -path ./vendor -prune -o -name "*go" -type f | grep -v vendor)
```

Comment:
nit: -name "*.go"
also isn't the `| grep -v vendor` redundant to the prune in find?

Reply:
The `| grep -v vendor` is required because the prune still includes the vendor dir (`./vendor/`) in the returned list.

We still want the prune though, because it makes it WAY faster.